Zend/zend_vm_def.h

```diff
@@ -3097,14 +3034,15 @@ ZEND_VM_HOT_NOCONST_HANDLER(43, ZEND_JMPZ, CONST|TMPVAR|CV, JMP_ADDR)
 	} else {
 		opline = OP_JMP_ADDR(opline, opline->op2);
 	}
-	FREE_OP1();
+	if (OP1_TYPE & (IS_TMP_VAR|IS_VAR)) {
```

Comment:
Is this going to work in the unspecialized executor? I'd expect `OP1_TYPE` to refer to the new opline there.